ionic-app/next.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  basePath: '',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/toost-net/toost-cloud/pullRequest?prid=8079923)
```suggestion
  basePath: "",
```